exonum/src/explorer/mod.rs

```diff
@@ -55,6 +55,25 @@ pub struct TxInfo {
     pub proof_to_block_merkle_root: ListProof<Hash>,
 }
 
+/// Range Information
```

Comment:
Nitpick: we (usually) ending comments by dot. :smile: Additionally, "information" should be in lower case.

Reply:
All right, I was not aware of the full-stop thing.

Reply:
Can you describe more detail? It is necessary to clarify that this is the range of blocks and describe whether the bound values are included in the range.